### ISSUE TITLE
salt-master at 100% cpu

### ISSUE THREAD

**Issue by mchugh19** [saltstack/salt#38866]
### Description of Issue/Question
salt-master seems responsive, but a single salt-master process is using 100% cpu. Problem occured on 2016.3.3 master. It was just upgraded to 2016.11.1, but with no difference. 

Old minion keys were deleted, but with no change. Problem occurs instantly on startup. 

```
PID USER      PR  NI    VIRT    RES    SHR S  %CPU %MEM     TIME+ COMMAND
12791 root      20   0  428552  52008   3496 R 100.0  0.7   3:49.96 salt-master
``` 

Event bus does not show any activity, and debug logs are also mostly empty after startup. 

### Steps to Reproduce Issue
Strace of the busy process just shows repeating
```
read(13, 0x384c1b3, 5)                  = -1 EAGAIN (Resource temporarily unavailable)
read(13, 0x384c1b3, 5)                  = -1 EAGAIN (Resource temporarily unavailable)
read(13, 0x384c1b3, 5)                  = -1 EAGAIN (Resource temporarily unavailable)
read(13, 0x384c1b3, 5)                  = -1 EAGAIN (Resource temporarily unavailable)
read(13, 0x384c1b3, 5)                  = -1 EAGAIN (Resource temporarily unavailable)
```

### Versions Report
```
Salt Version:
           Salt: 2016.11.1

Dependency Versions:
           cffi: 0.8.6
       cherrypy: 3.2.2
       dateutil: 1.5
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.7.2
        libgit2: 0.21.0
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.8
   mysql-python: Not Installed
      pycparser: 2.14
       pycrypto: 2.6.1
         pygit2: 0.21.4
         Python: 2.7.5 (default, Sep 15 2016, 22:37:39)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.3.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4

System Versions:
           dist: centos 7.2.1511 Core
        machine: x86_64
        release: 3.10.0-327.36.3.el7.x86_64
         system: Linux
        version: CentOS Linux 7.2.1511 Core
```

**Comment by gtmanfred** [comment]
Could you find out what that read is opening?  If you take the strace output, and walk backwards to find where the open for the `13` file handler in the read command is opening, we think it might become obvious what is going on.

Thanks!
Daniel

**Comment by mchugh19** [comment]
Perfect. Looked it up with `lsof -n -p<pid>`. Seemed to be the slack endpoint, and this issue was fixed in #38520

Adding in the sleep resolved the issue. Thanks!
